option java_package = "com.mi.game.network.data.bin";

//EnterZone 

message BINEnterZoneRequest {
	required int32 zoneId = 1;
}

message BINRoomConfig {
	required int32 roomGroupId = 1;
	optional string roomName = 2;
	required bool vipRoom = 3;
	required int32 minCash = 4;
	required int32 minGold = 5;
	required int32 minLevel = 6;
	required int32 roomCapacity = 7;
	required int32 playerSize = 8;
	required int32 minBet = 9;
	required int32 tax = 10;
	required bool active = 11;
}

message BINEnterZoneResponse {
    required bool responseCode = 1;
    optional string message = 2;
	optional bool enableDisplayRoomList = 3;
	repeated BINRoomConfig cashRoomConfigs = 4;
	repeated BINRoomConfig goldRoomConfigs = 5;
}

message BINExitZoneRequest {
	required int32 zoneId = 1;
}

message BINExitZoneResponse {
    required bool responseCode = 1;
    optional string message = 2;
}
//End EnterZone 
